server/config.ts

```diff
@@ -120,6 +127,7 @@ export default {
     },
     frontendComponents: {
       url: get('COMPONENT_API_URL', 'http://localhost:8083', requiredInProduction),
+      healthPath: '/ping',
```

Comment:
NOTE: [It's just `/ping` for Frontend Components API](https://frontend-components-dev.hmpps.service.justice.gov.uk/ping).